### ISSUE TITLE
Help debug faulty installation

### ISSUE THREAD

**Issue by isalirezag** [scikit-image/scikit-image#2793]
Does anyone know what is the problem with SSIM?
I cannot import neither find it.
I can successfully import skimage
but when i try

from skimage.measure import compare_ssim as ssim

it gives me this error:
ImportError: cannot import name compare_ssim

**Comment by stefanv** [comment]
What is the output of

```
import skimage
print(skimage.__version__)
```

**Comment by isalirezag** [comment]
0.13.0

**Comment by stefanv** [comment]
How about:

```
from skimage import measure
print(dir(measure))
```

**Comment by isalirezag** [comment]
['CircleModel', 'EllipseModel', 'LineModel', 'LineModelND', '__all__', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '__path__', '_ccomp', '_find_contours', '_find_contours_cy', '_label', '_marching_cubes_classic', '_marching_cubes_classic_cy', '_marching_cubes_lewiner', '_marching_cubes_lewiner_cy', '_marching_cubes_lewiner_luts', '_moments', '_moments_cy', '_pnpoly', '_polygon', '_regionprops', '_structural_similarity', 'approximate_polygon', 'block', 'block_reduce', 'compare_mse', 'compare_nrmse', 'compare_psnr', 'compare_ssim', 'correct_mesh_orientation', 'entropy', 'find_contours', 'fit', 'grid_points_in_poly', 'label', 'marching_cubes', 'marching_cubes_classic', 'marching_cubes_lewiner', 'mesh_surface_area', 'moments', 'moments_central', 'moments_hu', 'moments_normalized', 'perimeter', 'pnpoly', 'points_in_poly', 'profile', 'profile_line', 'ransac', 'regionprops', 'shannon_entropy', 'simple_metrics', 'structural_similarity', 'subdivide_polygon']
>>>

**Comment by isalirezag** [comment]
now that im trying:
ssim = measure.compare_ssim(A,A)
it sounds like working!!!

**Comment by SaraJ52** [comment]
how did you resolve the issue? I am having same

**Comment by O-K-Tiger** [comment]
I fixed it by reverting to scikit-image 0.16.2, 
pip install scikit-image==0.16.2

**Comment by JuanDValenciano** [comment]
if you use virtual environments with conda use:

`conda install -c conda-forge scikit-image`
`conda install -c conda-forge scikit-learn`
